client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -67,7 +79,7 @@ const IntroPricingBanner: FunctionComponent = () => {
 					/>
 					<span>
 						{ preventWidows(
-							translate( 'Get %(percent)d%% off your first year.', {
+							translate( 'Get up to %(percent)d%% off your first year.', {
```

Comment:
Very minor/optional suggestion, it might be nice to conditionally render the "Get up to x% off..." portion of the banner now that `highestDiscount` could potentially be `0`.

Reply:
I thought about this. Let me see if the banner will gracefully handle only having the 14-day money-back badge. Otherwise, we can leave the problem for when it becomes an issue.